services/svc-anchorSmoothScroll.js

```diff
@@ -0,0 +1,47 @@
+app.service('anchorSmoothScroll', function() {
```

Comment:
Buenísimo!
Podrías hacer el smooth un poco mas suave? Supongo que hay que demorarlo un poquito más, e idealmente darle easy-in/easy-out.

Reply:
Ahí fueron los dos cambios Tomi, avisame cómo lo ves!